schemas/0.1.0/profile.json

```diff
@@ -71,31 +76,41 @@
         "remark": {
           "type": "string",
           "title": "Remark",
-          "description": "Comment or guiding statement intended to be presented as supplementary information in user display."
+          "description": "Comment or guiding statement intended to be presented as supplementary information in user display.",
+          "minLength": 3,
```

Comment:
By making this a minLength of 3, does this mean that the `remark` is now required for validation?

Reply:
@jermnelson no - it means when it is _PRESENT_ it must have a minimum length of 3.  required would show up around line 15 - in the list of required properties.  :-)